extension/chrome/src/js/stash_page.js

```diff
@@ -2099,10 +2099,19 @@
 
 						// PR Filter
 						try {
-							// are we on the pull request list page ? raise exception if not
-							require('bitbucket/internal/feature/pull-request/table/pull-request-table');
+							// Are we on the pull request list page ? raise exception if not
+							try {
```

Comment:
you can also do this this:
```js
try {
  try {
    require('bitbucket/internal/feature/pull-request/table/pull-request-table');
  } catch (e1) {
    // Attempt to load legacy pull request list page
    require('bitbucket/internal/feature/pull-request/pull-request-table');
  }
} catch(e2) {
  // Attempt to load legacy pull request list page
  throw "Could not find resource 'pull-request-table'"
}
```
Or, I wonder, there might be simple way how to check bitbucket version

Reply:
Whoops I guess I was making comments on a review or something haha (they were not showing up/pending?); I agree with your simplified try/catch. Wish there was an easy way to check versions of bitbucket, but I suppose this will do just fine in the meantime :)